client/src/services/DonuService.ts

```diff
@@ -193,9 +192,31 @@ export const getModelInterface = async (model: Model.Model, selectedModelGraphTy
 
     const response = await callDonuCache(request);
     if (response.status === Donu.ResponseStatus.success) {
+      /** TODO
+       * This is a temporary mock up for Demonstration purposes.
+       * The DONU API needs to be clearer on which outputs can be selected programatically.
+       * Also, the HMI needs to be updated to allow the user to select which paramaters
+       * to use as `domain_parameter` for the simulation steps.
+       */
+      if (selectedModelGraphType === Model.GraphTypes.FunctionNetwork) {
+        const result = response?.result as Donu.ModelDefinition;
+        let outputs = [];
+        let domainParameter;
+        if (modelGraph.model === 'SimpleSIR_metadata_gromet_FunctionNetwork.json') {
+          outputs = ['P:sir.out.S', 'P:sir.out.I', 'P:sir.out.R'];
```

Comment:
Could we just have `S, I, R` for these variables?

Reply:
I can hard code in the backend to rename them visually if that's what you are saying.
Those are `uid` and I cannot change them, but I can set their `metadata.name` based on what you want.

Reply:
Yep, that is what I meant.